{"name":"feature","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.StringIndexOutOfBoundsException: begin 0, end 58, length 25","trace":"cucumber.runtime.CucumberException: java.lang.StringIndexOutOfBoundsException: begin 0, end 58, length 25\n\tat cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)\n\tat com.fblogin.testrunner.TestRunners.feature(TestRunners.java:38)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\nCaused by: java.lang.StringIndexOutOfBoundsException: begin 0, end 58, length 25\n\tat java.base/java.lang.String.checkBoundsBeginEnd(String.java:3734)\n\tat java.base/java.lang.String.substring(String.java:1903)\n\tat com.fblogin.tests.testcases.TestCases.pageValidate(TestCases.java:42)\n\tat com.fblogin.stepsdefinition.FacebookStepDefinition.it_shall_redirect_to_login_page(FacebookStepDefinition.java:38)\n\tat âœ½.Then It shall redirect to login page(FacebookLogin.feature:15)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg0","value":"Facebook wrong password login failure validation"}],"start":1621708928085,"stop":1621708940706,"uuid":"5edf6313-6eb9-44cb-8838-b8ced366f190","historyId":"7de3436529f22bd50b51ed9e5eafd264","fullName":"com.fblogin.testrunner.TestRunners.feature","labels":[{"name":"package","value":"com.fblogin.testrunner.TestRunners"},{"name":"testClass","value":"com.fblogin.testrunner.TestRunners"},{"name":"testMethod","value":"feature"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"/Users/kehindeadeoya/Documents/projects/FBLogin/src/com/fblogin/testrunner/TestRunner.java"},{"name":"subSuite","value":"com.fblogin.testrunner.TestRunners"},{"name":"host","value":"wikiwoos-MacBook-Pro.local"},{"name":"thread","value":"4820@wikiwoos-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}